src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.json

```diff
@@ -439,6 +439,9 @@
         },
```

Comment:
Related to nothing specific, but I like that you do these as "micro-PR's", makes it much more manageable for reviewers.